### DISCUSSION TITLE
Is it normal for a search commant to take almost a minute to complete?

### DISCUSSION THREAD

**Discussion by shinayser** [microsoft/winget-cli#1810] [General]
I just started to work with winget and well, it is takeing a really logne time to complete its commands. Search, for isntance, sometimes takes 30-40 seconds to complete.

This picture below is of me performing `winget search less`

<img width="460" alt="image" src="https://user-images.githubusercontent.com/1146982/146451202-ff0c5c46-760e-4559-8d68-5e2f2a342618.png">

When I run it again, it returns really fast, but the first time each command takes a looooong time to complete.
There is anything else I can do to improve it?

**Comment by ghost** [comment]
Because `winget search` depends on `winget list` command and they are **_reluctant_** to fix `winget list` in the short term. see [Winget List command takes 10 seconds to complete. #855](https://github.com/microsoft/winget-cli/issues/855)
